x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -54,6 +54,10 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
         serverArgs: [
           ...xPackApiIntegrationTestsConfig.get('kbnTestServer.serverArgs'),
           '--xpack.actions.enabled=true',
+          `--xpack.actions.whitelistedHosts=${JSON.stringify([
+            'localhost',
```

Comment:
Interesting, I guess we have to include `localhost` here for our simulators!  I'm wondering if it's going to be a pain that `localhost` isn't default whitelisted - probably not much of one, and there is a security angle even for accessing localhost services.

Reply:
Yeah, I don't think there's a way around this without hardcoding localhost and I'd rather not do that unless we get some concrete need for it.